static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -140,18 +137,13 @@ function WidgetViewerModal(props: Props) {
   ].includes(widget.displayType);
 
   if (shouldReplaceTableColumns) {
-    tableWidget.queries[0].orderby = tableWidget.queries[0].orderby || '-timestamp';
-    fields.splice(
-      0,
-      fields.length,
-      ...['title', 'event.type', 'project', 'user.display', 'timestamp']
-    );
+    if (fields.length === 1) {
```

Comment:
why do we only add an order by if `fields.length === 1`?

Reply:
(ie should be be consistent and just not add orderbys?)

Reply:
Hey Shruthi, for queries with multiple fields like below, I didn't add an orderby since it's a bit arbitrary to pick just one of the fields to order:
![image](https://user-images.githubusercontent.com/83961295/158458757-64c68177-5edf-40d6-aadb-5ae0c4578292.png)

For queries with just one field like below (besides the title), it felt more natural to just order by the single field automatically.
![image](https://user-images.githubusercontent.com/83961295/158458889-e3d46ed2-05b3-4aa2-afa8-0e423d2857d1.png)

I'll check with Jasmin if we should just not do any orderbys and update in a followup pr if that sounds good to you!

Reply:
Yup, sounds good!